docs/microsoft_entra_workload_id.md

```diff
@@ -0,0 +1,555 @@
+OpenShift credentials provisioning tool
+Usage:
+  ccoctl [command]
+
+Available Commands:
+  aws         Manage credentials objects for AWS cloud
+  azure       Manage credentials objects for Azure
+  completion  Generate the autocompletion script for the specified shell
+  gcp         Manage credentials objects for Google cloud
+  help        Help about any command
+  ibmcloud    Manage credentials objects for IBM Cloud
+  nutanix     Manage credentials objects for Nutanix
+
+Flags:
+  -h, --help   help for ccoctl
+
+Use "ccoctl [command] --help" for more information about a command.
+```
+
+### Step 2: Creating Azure resources with the ccoctl tool
+
+See: [OpenShift Documentation - Creating Azure resources with the ccoctl tool](https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html-single/installing_on_azure/index#cco-ccoctl-creating-at-once_installing-azure-customizations)
+
+#### 2.1 Extract Credentials Requests from OpenShift Release Image
+
+Extract the CredentialsRequest objects from the release image:
+
+```bash
+$ oc adm release extract --cloud=azure --credentials-requests $RELEASE_IMAGE --to=credrequests
+```
+
+Output:
+
+```bash
+$ ls credrequests  
+0000_26_cloud-controller-manager-operator_14_credentialsrequest-azure.yaml
+0000_50_cluster-image-registry-operator_01-registry-credentials-request-azure.yaml  
+0000_50_cluster-storage-operator_03_credentials_request_azure.yaml
+0000_30_cluster-api_01_credentials-request.yaml
+0000_50_cluster-ingress-operator_00-ingress-credentials-request.yaml                
+0000_50_cluster-storage-operator_03_credentials_request_azure_file.yaml  
+0000_30_machine-api-operator_00_credentials-request.yaml
+0000_50_cluster-network-operator_02-cncc-credentials.yaml
+```
+
+#### 2.2 Log in to Azure CLI
+
+To enable the ccoctl utility to detect your Azure credentials automatically, log in to the Azure CLI:
+
+```bash
+$ az login
+```
+
+Below is an example of the Azure credentials file:
+
+```bash
+$ cat <azure_credentials_file_path>/osServicePrincipal.json
+{
+  "subscriptionId": "REDACTED",
+  "clientId": "REDACTED",
+  "clientSecret": "REDACTED",
+  "tenantId": "REDACTED"
+}
+```
+
+Output:
+
+```bash
+$ az login
+A web browser has been opened at https://login.microsoftonline.com/organizations/oauth2/v2.0/authorize. Please continue the login in the web browser. If no web browser is available or if the web browser fails to open, use device code flow with `az login --use-device-code`.
+The following tenants don't contain accessible subscriptions. Use 'az login --allow-no-subscriptions' to have tenant level access.
+xxx 'Red Hat, Inc'
+xxx 'PnT Cloud Ops Dev/Test'
+[
+  {
+    "cloudName": "AzureCloud",
+    "homeTenantId": "REDACTED",
+    "id": "REDACTED",
+    "isDefault": true,
+    "managedByTenants": [],
+    "name": "REDACTED",
+    "state": "Enabled",
+    "tenantId": "REDACTED",
+    "user": {
+      "name": "REDACTED",
+      "type": "REDACTED"
+    }
+  }
+]
```

Comment:
Interesting, I do not get this output on `az login`, but need to do `az account show` to see this. It's fine like this though, we don't really use this info in the rest of the doc.